stable/postgresql/README.md

```diff
@@ -212,6 +215,25 @@ With NetworkPolicy enabled, traffic will be limited to just port 5432.
 For more precise policy, set `networkPolicy.allowExternal=false`. This will only allow pods with the generated client label to connect to PostgreSQL.
 This label will be displayed in the output of a successful install.
 
+## Deploy chart using Docker Official PostgreSQL image
```

Comment:
image -> Image (capitalized as per the official docs https://docs.docker.com/docker-hub/official_images/)